docs/quickstart/java_api_quickstart.md

```diff
@@ -111,7 +111,7 @@ In IntelliJ IDEA recommended way to change JVM options is from the `Help | Edit
 ## Build Project
 
 If you want to __build/package your project__, go to your project directory and
-run the '`mvn clean package`' command.
```

Comment:
I think it actually looks nicer with the ticks. https://ci.apache.org/projects/flink/flink-docs-master/quickstart/java_api_quickstart.html#build-project

Will revert this while merging.